src/Normalizer/Pap/AppendCampaignStatsNormalizer.php

```diff
@@ -43,6 +47,8 @@ public function normalize($object, $format = null, array $context = [])
 
         $campaign['nb_surveys'] = $object->getCampaignHistoriesWithDataSurvey()->count();
         $campaign['nb_visited_doors'] = $this->campaignHistoryRepository->countVisitedDoors($object);
+        $campaign['nb_addresses'] = $object->getNbAddresses();
```

Comment:
we can use `Group("pap_campaign_read")` on getter for do the same job

Reply:
We need to return this info only for Jemengage Admin, but using a group `pap_campaign_read` will return this info also for app mobile.